client/src/templates/Challenges/components/test-suite.tsx

```diff
@@ -20,47 +20,37 @@ interface TestSuiteProps {
   tests: Test[];
 }
 
-function getAccessibleText(text: string, err?: string, pass?: boolean) {
-  let accessibleText = 'Waiting';
-  const cleanText = text.replace(/<\/?code>/g, '');
-
-  // Determine test status (i.e. icon)
-  if (err) {
-    accessibleText = 'Error';
-  } else if (pass) {
-    accessibleText = 'Pass';
-  }
-
-  // Append the text itself
-  return accessibleText + ' - ' + cleanText;
-}
-
 function TestSuite({ tests }: TestSuiteProps): JSX.Element {
   const testSuiteTests = tests.filter(isTestSuiteTest);
 
   return (
-    <div className='challenge-test-suite'>
-      {testSuiteTests.map(({ err, pass = false, text = '' }, index) => {
-        const isInitial = !pass && !err;
-        const statusIcon = pass && !err ? <GreenPass /> : <Fail />;
-        return (
-          <div
-            aria-label={getAccessibleText(text, err, pass)}
-            className='test-result'
-            key={text.slice(-6) + String(index)}
-          >
-            <div className='test-status-icon'>
-              {isInitial ? <Initial /> : statusIcon}
-            </div>
-            <div
-              aria-hidden='true'
-              className='test-output'
-              dangerouslySetInnerHTML={{ __html: text }}
-            />
-          </div>
-        );
-      })}
-    </div>
+    <>
+      <h2 className='challenge-test-suite-heading'>Tests</h2>
+      <ul className='challenge-test-suite'>
+        {testSuiteTests.map(({ err, pass = false, text = '' }, index) => {
+          const isInitial = !pass && !err;
+          const statusIcon = pass && !err ? <GreenPass /> : <Fail />;
+          const initialText = 'Waiting';
+          const statusText = pass && !err ? 'Passed' : 'Failed';
```

Comment:
Maybe outside the scope of this PR, but shouldn't we put these in the translations?

Reply:
added